src/js/product-details.js

```diff
@@ -87,23 +113,23 @@ const canonicalProducts = {
   resources: [
     {
       title: 'Webinars',
-      url: 'https://blog.ubuntu.com/?category=webinars#posts-list',
+      url: 'https://www.ubuntu.com/blog?category=webinars#posts-list',
```

Comment:
Can you update this to `https://ubuntu.com/blog` to save a redirect.

Reply:
@anthonydillon there are a few other www.ubuntu.com links in there, not related to /blog - do you want them updated in the same way?